R/utils-classes.R

```diff
@@ -316,7 +315,10 @@ show_query.bcdc_promise <- function(x, ...) {
 #' }
 show_query.bcdc_sf <- function(x, ...) {
 
-  url <- url_format(attributes(x)$url)
+  url <- attributes(x)$url
```

Comment:
Another way is to use `attr()`, though I don't think it really matters which...

```suggestion
  url <- attr(x, "url")
```